### PR TITLE
Patch 4

### PR DESCRIPTION
**Pull request by clicklin**
Please apply patch 4.

### CODE REVIEW

File: views/templates/hook/admin_order.tpl [hunk 1]
```diff
@@ -104,6 +115,12 @@
                             {elseif $clorder['hour_range_from'] eq '4'}
                                 15:00 - 17:30
                             {/if}
+                            {elseif $clorder['hour_range_from'] eq '5'}
+                                9:00 - 14:00
+                            {/if}
+                            {elseif $clorder['hour_range_from'] eq '6'}
+                                14:00 - 18:00
+                            {/if}
```
Comment (Quetzacoalt91) [inline views/templates/hook/admin_order.tpl]: There are syntax errors here: 
```javascript
{elseif $clorder['hour_range_from'] eq '6'}
	14:00 - 18:00
{/if}
```
the two previous {/if} should not be written.

**Comment by clicklin** [comment]
hi, last week we change this syntax error, can you merge? Thx

File: views/templates/hook/admin_order.tpl [hunk 2]
```diff
@@ -148,10 +163,12 @@
                         <label class="control-label col-lg-3">{l s='Hour range from' mod='clickline'}: <span style="color:red;">*</span></label>
                         <div class="col-lg-2">
                             <select name="hour_range_from">
-                                <option value="1" {if $params['hour_range_from'] eq '1'}selected="selected"{/if}>09:00 - 11:00</option>
+                                <!--<option value="1" {if $params['hour_range_from'] eq '1'}selected="selected"{/if}>09:00 - 11:00</option>
```
Comment (gaillafr) [inline views/templates/hook/admin_order.tpl]: Hi @clicklin, instead of commenting this section, would it be possible to just remove it?
Thank you.

**Comment by clicklin** [comment]
Hi, we delete the source comments

**Comment by gaillafr** [comment]
Thank you @clicklin, your PR has been merged.
Kind regards.

**Pull request merged**